Lombiq.HelpfulLibraries.Common/Extensions/StringExtensions.cs

```diff
@@ -238,6 +240,26 @@ public static IEnumerable<int> AllIndexesOf(this string text, string value)
         }
     }
 
+    /// <summary>
+    /// Turns a camelCase or PascalCase token into snake_Case.
+    /// </summary>
+    /// <param name="input">The input in camelCase or PascalCase.</param>
+    /// <returns>
+    /// The input converted to snake_Case. It doesn't alter case so you can call either ToUpper or ToLower without any
+    /// additional penalties.
+    /// </returns>
+    /// <remarks>
+    /// <para><see href="https://stackoverflow.com/a/18781533"/>.</para>
+    /// </remarks>
+    public static string ToSnakeCase(this string input) =>
+        string.Concat(input.Select((character, index) => index > 0 && char.IsUpper(character) ? "_" + character : character.ToString()));
+
+    /// <summary>
+    /// Returns <see langword="true"/> if the value is <see langword="true"/> or <c>True</c>.
+    /// </summary>
+    [SuppressMessage("Minor Code Smell", "S4225:Extension methods should not extend \"object\"", Justification = "No.")]

```

Comment:
Ah yes, my favorite justification.
![image](https://user-images.githubusercontent.com/4007293/160364439-a763254b-e560-47b0-a141-02444cb4ac29.png)

Reply:
😁

Reply:
Not coincidentally, this code comes from you :D. I just copied it over.

Reply:
Indeed.
![image](https://user-images.githubusercontent.com/4007293/160395174-5994bcbb-ac11-4235-bab1-9c17dbcf2c4a.png)

Reply:
Thinking about it, this is an extremely specific use-case. So rather, I've moved it to `Gui`.